source/api/passwords.md

```diff
@@ -123,7 +123,7 @@ To customize the contents of the email, see
 
 This is an `Object` with several fields that are used to generate text/html
 for the emails sent by `sendResetPasswordEmail`, `sendEnrollmentEmail`,
-and `sendVerificationEmail`.
+and `sendVerificationEmail`. Their respective contents can be set by setting the `resetPassword`, `enrollAccount` and `verifyEmail` fields on the `Object`.
```

Comment:
Isn't this made clear in the bulleted list directly below this?  Also, it's not just setting the fields, the properties themselves should be `Object`s, which again seems to be demonstrated below?